packages/is-shallow-equal/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/is-shallow-equal",
-	"version": "2.3.0",
+	"version": "3.0.0",
```

Comment:
Here as well, it should remain unchanged. Lerna will bump version at the time of npm publishing. 

How does it work in Calypso, I'm curious 😃

Reply:
Ah I should have read the link at the top of the file 😉 In Calypso these are maintained manually as far as I know!